Core/AppUrls.swift

```diff
@@ -22,26 +22,29 @@ import Foundation
 public struct AppUrls {
 
     private struct Url {
-
-        // You can change this to use a subdomain for testing (e.g. "test.")
-        static let subdomain = ""
```

Comment:
We never used this and now that our instances are behind security probably won't.